### PR TITLE
Baylee Review (Nov 12th)

### PR DESCRIPTION
**Pull request by angelicahill**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -5,62 +5,125 @@ import (
...
-	Scanner.Scan()
-	userInputWord := Scanner.Text()
-	userInputLowercase := strings.ToLower(userInputWord)
+	log.Fatal(http.ListenAndServe(":8080", http.FileServer(http.Dir("static"))))
 
-	timeAtStart := time.Now()
+	fmt.Println("Hello and welcome to the Shakespeare Scanner.\n This is a tool which allows you to search a Shakespeare play for a word\n and it will tell you both how many times it shows up, as well as where it shows up.\n")
+	for {
+		fmt.Println("Please type the title of the play you would like to search for your word...")
+		Scanner := bufio.NewScanner(os.Stdin)
```
Comment (baylee) [inline main.go]: Probably more idiomatic to use lowercase `scanner`

File: main.go [hunk 2]
```diff
@@ -5,62 +5,125 @@ import (
...
-	fmt.Printf("Please pick a word and then I will tell you how often is shows up in each of the following Shakespeare plays: The Tempest, Hamlet, King Lear, Macbeth, and Romeo and Juliet. Please pick a word...")
-	Scanner := bufio.NewScanner(os.Stdin)
-	Scanner.Scan()
-	userInputWord := Scanner.Text()
-	userInputLowercase := strings.ToLower(userInputWord)
+	log.Fatal(http.ListenAndServe(":8080", http.FileServer(http.Dir("static"))))
 
-	timeAtStart := time.Now()
+	fmt.Println("Hello and welcome to the Shakespeare Scanner.\n This is a tool which allows you to search a Shakespeare play for a word\n and it will tell you both how many times it shows up, as well as where it shows up.\n")
+	for {
```
Comment (baylee) [inline main.go:34]: Nice use of this loop to stay in the program

File: main.go [hunk 3]
```diff
@@ -5,62 +5,125 @@ import (
...
+			}
+		}
+		Scanner = bufio.NewScanner(os.Stdin)
+		Scanner.Scan()
+		finalWord := strings.ToLower((Scanner.Text()))
+		fmt.Printf("Ok so just to confirm the word you want to search is %s.\n Now searching...\n", finalWord)
+		getPlay, err := ioutil.ReadFile(userPlayChoice + ".txt")
+		if err != nil {
+			fmt.Println("Sorry we do not currently have that play in our database, please try another play.\n")
+			return
```
Comment (baylee) [inline main.go]: Any reason why you use `return` in some places and `break` in others to exit the loop?

File: main.go [hunk 4]
```diff
@@ -5,62 +5,125 @@ import (
...
+		}
+		playString := string(getPlay)
```
Comment (baylee) [inline main.go]: Seems like all of this bit could be its own function for legibility and testability

**Review by baylee — APPROVED** [review]
So, to make sure I'm following along, the key pieces in this PR are:
1. Work with a larger set of plays and only scan the user-selected one (instead of hard-coding the goroutines for each play as before)
2. Allowing the program to loop and generally be more interactive
3. Serve a static HTML page

Nice work! I'm going to create some Issues in Github with some suggested next features.

File: main.go (hunk 1 above)
Comment (angelicahill) [reply main.go]: Will edit thank you :)

File: main.go (hunk 2 above)
Comment (angelicahill) [reply main.go:34]: Thank you 👍

File: main.go (hunk 3 above)
Comment (angelicahill) [reply main.go]: Thank you for pointing this out. I went back and made changes to rectify my use of "return", "break" and "continue" so they are used correctly.

File: main.go (hunk 4 above)
Comment (angelicahill) [reply main.go]: I extracted the functionality that was segmenting the .txt file into acts (using roman numerals) and put it into a the following function: `'sortingActs(getPlay []byte) []string` 

Is that what you were suggesting, or did you also think I should include the functionality that ranges over the acts and counts the words (line 56 - 65) into this, or another function?

**Comment by angelicahill** [comment]
@baylee I would really appreciate you taking another look at this when you have time. Thank you :)

File: main.go (hunk 4 above)
Comment (baylee) [reply main.go]: the `sortingActs` function is what I was imagining -- looks good.

Comment (angelicahill) [reply main.go]: great thanks 👍

**Pull request merged**